ckanext/ontario_theme/templates/external/home/snippets/ontario_theme_contact_us.html

```diff
@@ -4,19 +4,14 @@
 
 #}
 
-<div class="contact-us new-alert alert-contact alert-clear">
-    <div class="container">
-      <div class="row">
-        <div class="col-md-2 symbol">
-          <i class="fa fa-comments"></i>
-        </div>
-        <div class="col-md-10 message">
-          <p>
-            {% trans %}Have feedback or questions? Have a data or catalogue request? We want to hear from you! We use your feedback to drive our work.{% endtrans %}
-            <br />
-            <a href="{% trans %}https://www.ontario.ca/feedback/contact-us?id=532365&nid=138269{% endtrans %}" class="btn btn-secondary float-right">{{ _("Contact Us") }}!</a>
-          </p>  
-        </div>
+<div class="feedback ontario-alert ontario-alert--informational">
+  <div class="feedback ontario-alert__header">
+      <div class="feedback ontario-alert__header-icon">
+            <i class="fa fa-comments"></i>
       </div>
-    </div>
-</div>
+  </div>
+  <div class="feedback ontario-alert__body">
+    <h2 class="feedback ontario-alert__header-title ontario-h4">{% trans %}Help us improve the Data Catalogue{% endtrans %}</h2>
```

Comment:
The `ontario-h4` within an `h2` tag is from the Design System itself, so we can create something like `data-h2`, if needed because the DS font and typography will be added with all the ontario-h* classes at some point. This may create more work implementing DS components in the future though.

Reply:
Ok we'll leave it at that.